e2e-tests/src/tests/studies/ourhealth/new-registration-age18.test.ts

```diff
@@ -0,0 +1,93 @@
+import { expect } from '@playwright/test'
+import { test } from 'lib/fixtures/ourhealth-fixture'
+import * as process from 'process'
+import { Study } from 'src/data/constants'
+import StudyConsent from 'pages/ourhealth/study-consent'
+import StudyCreateAcct from 'pages/ourhealth/study-create-acct'
+import StudyDashboard, { Activities } from 'pages/ourhealth/study-dashboard'
+import { emailAlias, goToStudyEligibility } from 'tests/e2e-utils'
+
+test.describe('Home page', () => {
```

Comment:
[video.webm](https://github.com/broadinstitute/juniper/assets/35533885/f0dcf94c-043a-4235-9d45-0390ba99375c)

video playback speed can be changed (slow down) down by click the three-dots icon on the bottom left.